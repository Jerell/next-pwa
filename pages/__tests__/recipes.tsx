// import { render, screen } from '@testing-library/react'
import Recipes from "../recipes"
import { mount, shallow, render } from 'enzyme';


describe('<Recipes/>', () => {
  it("should render heading", () => {
    const component = shallow(<Recipes />)
    expect(component).toMatchSnapshot()
  })
})
