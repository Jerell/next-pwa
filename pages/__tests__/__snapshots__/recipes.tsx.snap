// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Recipes/> should render heading 1`] = `
<Page>
  <section
    className="mt-20"
  >
    <h2
      className="text-xl font-semibold"
    >
      Ingredients
    </h2>
    <p
      className="mt-2 text-gray-600 dark:text-gray-400"
    >
      Like any good recipe, we appreciate community offerings to cultivate a delicous dish.
    </p>
  </section>
  <section
    className="mt-10"
  >
    <h3
      className="font-medium"
    >
      Thanks to
    </h3>
    <ul
      className="space-y-2 px-6 py-2 text-sm text-gray-600 dark:text-gray-400 list-disc"
    >
      <li>
        <a
          className="underline"
          href="https://unsplash.com"
        >
          Unsplash
        </a>
         
        for high quality images
      </li>
      <li>
        <a
          className="underline"
          href="https://teenyicons.com"
        >
          Teenyicons
        </a>
         
        for lovely icons
      </li>
    </ul>
  </section>
</Page>
`;
